packages/python/plotly/plotly/py.typed

```diff
@@ -0,0 +1 @@
+partial
```

Comment:
`py.typed` is just a marker file, per https://peps.python.org/pep-0561/#packaging-type-information

> This marker applies recursively

It should be blank (e.g. just a file without any contents), so can you remove the text "partial"?

---

Are you interpreting the text `partial` to be like a signifier of partial typing? I don't think anyone actually consumes the content of this file, for example `mypy` just goes based on the file existing: https://github.com/python/mypy/blob/v1.14.1/mypy/modulefinder.py#L344-L345

Reply:
Oh I follow what you mean, thanks for sharing. TIL that `partial` means to first check a stub package, then fall back on the runtime package.

Though I still don't think `partial` should be here, as this is not a stub package, this is the actual runtime package. Does that make sense?

In other words, if this was an external stub package filled with `.pyi`, then you'd put `partial`. But that's not the case here, it's the runtime package with `.py` files, so `py.typed` should be blank

Reply:
I see. Fixed.